lib/trackler/problem.rb

```diff
@@ -6,15 +6,16 @@ class Problem
     attr_reader :slug, :root
```

Comment:
The `attr_reader` for `:root` will not work properly any more. 
The easy fix it to just define the `@root` instance variable.
Given it seems to be untested it might not be used externally and should be removed, but it would be useful to double check this.